components/camel-zipkin/src/main/java/org/apache/camel/zipkin/ZipkinClientRequestAdapter.java

```diff
@@ -56,5 +56,15 @@ void onRequest(Exchange exchange, SpanCustomizer span) {
                 cache.reset();
             }
         }
+        
+        Object clientTagsObj = exchange.getProperty("camel.custom.tags");
```

Comment:
Cant we use `camel.client.customtags` eg with `camel.client` as prefix.

And then you can use `exchange.getProperty("xxx", Map.class);

Reply:
Have updated property name to camel.client.customtags and used getProperty() method with class type.